api/lib/domain/errors.js

```diff
@@ -6,6 +6,24 @@ class DomainError extends Error {
   }
 }
 
+class LocaleFormatError extends DomainError {
+  constructor(locale) {
+    super();
+    this.message = `Given locale is in invalid format: "${locale}".`;
+    this.code = 'INVALID_LOCALE_FORMAT';
+    this.meta = { locale };
+  }
+}
+
+class LocaleNotSupportedError extends DomainError {
+  constructor({ locale }) {
```

Comment:
Suite à notre dernière mob ensemble on n'utilise plus d'autres propriétés que `locale` : 

```suggestion
  constructor(locale) {
```